### PR TITLE
fixes #412 make fetch error messages simpler

### LINKED ISSUE #412: make fetch error messages more detailed
Currently 405s don't have a response body so nothing gets shown. In this case we could show our own error string along with the status code.

### PR DESCRIPTION
**Pull request by spasovski**
image

### CODE REVIEW

File: src/state/api.js [hunk 1]
```diff
@@ -28,26 +29,17 @@ export async function getProbeData(params, token) {
...
-      const txt = await response.text();
-      e.moreInformation = `GLAM encountered a problem trying to show ${
-        params.probe
-      }, so we'll
-      report the error here:
-      ${txt.split('\n').slice(0, 2)}`;
-      throw e;
+    // Catch fetch error responses and show them in the UI.
+    if (response.status >= 400 && response.status < 600) {
+      const errorCode = 'code' + response.status;
```
Comment (openjck) [inline src/state/api.js]:
```
  34:25  error  Unexpected string concatenation  prefer-template
```

Comment (spasovski) [reply src/state/api.js]: That's a little strict, simple concatenations shouldn't need it. Don't mind changing it just weird.

Comment (openjck) [reply src/state/api.js]: Yeah, Airbnb rules. 🤷 

We could change that rule if we prefer.

**Review by openjck — CHANGES_REQUESTED** [review]
Looks good to me other than the eslint thing. 👍

File: src/state/api.js (hunk 1 above)
Comment (hamilton) [reply src/state/api.js]: I do prefer string templating in cases like this

**Pull request merged**